src/modules/contest/contest.service.ts

```diff
@@ -76,7 +80,138 @@ export class ContestService {
         },
       ],
       where: { id: contestId },
+      rejectOnEmpty: true,
```

Comment:
why use `rejectOnEmpty`? it will throw 500 when not found

Reply:
Fixed